internal/config/config.go

```diff
@@ -19,28 +19,33 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-type Tokens struct {
-	GithubToken string `yaml:"githubToken"`
-	JiraToken   string `yaml:"jiraToken"`
+type Config struct {
+	JiraBaseURL string `yaml:"jiraBaseURL"`
```

Comment:
IMO, this should have the omitEmpty notation on this, with a suitable default value used if not provided.